src/App.tsx

```diff
@@ -8,6 +8,7 @@ import {
 import dayjs from "dayjs";
 import utc from "dayjs/plugin/utc";
 
+import {CssVarsProvider} from "@mui/joy/styles";
```

Comment:
```suggestion
import CssVarsProvider from "@mui/joy/styles/CssVarsProvider";
```

[Tree-shaking](https://webpack.js.org/guides/tree-shaking/) does not always work so it's better to explicitly import the needed stuff to avoid importing stuff we don't use.